src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java

```diff
@@ -973,15 +973,11 @@ public void run() {
 							logger.debug("Getting latest version info from " + longUrl);
 				    		try {
 				    			latestVersionInfo = new AddOnCollection(getRemoteConfigurationUrl(longUrl), getPlatform(), false);
-				    		} catch (SSLHandshakeException e2) {
+				    		} catch(SSLHandshakeException | InvalidCfuUrlException e2) {
 					    		if (callback != null) {
 					    			callback.insecureUrl(longUrl, e2);
 					    		}
-							} catch (InvalidCfuUrlException e2) {
-					    		if (callback != null) {
-					    			callback.insecureUrl(longUrl, e2);
-					    		}
-							} catch (Exception e2) {
+							} catch(Exception e2) {
```

Comment:
Dear @thc202, I will proceed according to your comments and (as soon as possible) I will update this pull request. Thanks!